PasswordVerifier/src/com/company/PasswordVerifier.java

```diff
@@ -0,0 +1,81 @@
+public class PasswordVerifier {
+
+    public static void main(String[] args) {
+        String pass = "abdsdscdEF";
+        System.out.println("Password is : "+pass);
+        try {
+            System.out.println("Response is : "+verify(pass));
+        } catch (InvalidPasswordException e) {
+            System.out.println("Response is : "+e.getMessage());
+        }
+    }
+
+    public static String verify(String pass) throws InvalidPasswordException {
```

Comment:
I'd rename this variable to `password`. The four characters you save are not worth the potential confusion.